cps/main.c

```diff
@@ -672,6 +678,65 @@ fill_bgp_rule(struct ipv6_acl_rule *rule, struct gatekeeper_if *iface,
 	}
 }
 
+/*
+ * Match the packet if it fails to be classifed by ACL rules,
+ * if it's bgp packet, then submit it to the LLS block.
```

Comment:
Touched-up sentence: `If it's a bgp packet, then submit it to the LLS block.` I capitalized it and added an "a".